particles/Greeting/Greeting.js

```diff
@@ -0,0 +1,58 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+"use strict";
+
+// Greeting defines a particle that displays "You're an {{animal}}" for a
+// given Person entity. The animal is chosen based on the first letter of
+// the person's name.
+defineParticle(({DomParticle}) => {
+
+  let template = `
```

Comment:
🚲 🏠  ("bikeshed", IOW I'm expressing a weak subjective thought here so feel free to take no action if you feel differently).

Prefer `const` to `let` when you know the identifier won't be re-assigned.